xmtp_mls/src/groups/group_membership.rs

```diff
@@ -26,6 +26,7 @@ impl GroupMembership {
         self.members.get(inbox_id.as_ref())
     }
 
+    #[allow(clippy::unnecessary_get_then_check)]
     pub fn diff<'inbox_id>(
         &'inbox_id self,
         new_group_membership: &'inbox_id Self,
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 54]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->You mind just switching this to `contains_key`? Then we don't need the clippy allow<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/xmtp/libxmtp/703?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
Switched 👍